Makefile

```diff
@@ -45,7 +45,7 @@ prepare:
 	cp $(SPEC_FILE) $(BUILD_DIR)/SPECS/
 
 rpm_package_source:
-	tar --transform 'flags=r;s,^,/${NAME}-${VERSION}/,' --exclude .git --exclude dist -cvjf $(SOURCE_PATH) .
+	tar --transform 'flags=r;s,^,/${NAME}-${VERSION}/,' -cvjf $(SOURCE_PATH) $(SPEC_FILE) --exclude .git --exclude "*.spec" --exclude dist .
```

Comment:
This ensures that exactly one `*.spec` file is packaged with RPM source. If there more then 1 such file, `rpmbuild` will complain about multiple spec files. This change ensure we package the same spec file which was also copied to `(BUILD_DIR)/SPECS/`.

Reply:
Makes sense.